charts/thoras/values.yaml

```diff
@@ -60,6 +60,19 @@ thorasApiServer:
   port: 443
   logLevel: "debug"
 
+thorasApiServerV2:
+  enabled: false
+  containerPort: 8443
+  podAnnotations: {}
+  limits:
+    cpu: 1000m
+    memory: 1000Mi
+  requests:
+    cpu: 100m
```

Comment:
[microbes] Just sharing that I read somewhere that using powers of 2 for these types of k8s settings can actually makes things fit better in the end, on top of being more aesthetically pleasing (so `128m` in this instance) -- not sure how much BS that is but I always do it cuz I agree it looks more professional and who knows maybe it does fit better why not